docs/diagrams/AddClaimSequenceDiagram.puml

```diff
@@ -0,0 +1,87 @@
+@startuml
```

Comment:
This can be broken into smaller sequence diagrams as there are multiple actions within this diagram